microstoragetest/storage.go

```diff
@@ -0,0 +1,18 @@
+package microstoragetest
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/microstorage"
+	"github.com/giantswarm/microstorage/memory"
+)
+
+// Must creates a configured storage ready to be used in tests.
+func Must() microstorage.Storage {
```

Comment:
`MustNew` would be the right thing, wouldn't it? Don't want to stress further on this though.